android/runtime/common/src/java/org/appcelerator/kroll/util/KrollAssetHelper.java

```diff
@@ -38,6 +39,7 @@
 
 	public interface AssetCrypt {
 		InputStream openAsset(String path);
+		byte[] readAssetBytes(String path);
```

Comment:
Adding a method to the `AssetCrypt` interface will cause our "appc-verify" module to crash since it implements it. And we distribute "appc-verify" via the CLI, so, I'd prefer to avoid this. (I also remember some other library implementing this interface too, but I don't remember which.)

Because of this, let's not add a method to this interface. However, we should definitely keep the `KrollAssetHelper.readAssetBytes()` method, which can create the `byte[]` via the interface's `openAsset()` returned `InputStream`.

What if we changed `AssetCryptImpl.getAssetBytes()` to return an `InputStream` type (really the `CypherInputStream`) instead of `byte[]`? Then its `openAsset()` method implementation would like this...
```javascript
@Override
public InputStream openAsset(String path)
{
	return AssetCryptImple.getAssetBytes(path);
}
```

Reply:
Resolved with https://github.com/appcelerator/titanium_mobile/pull/11045/commits/d38282b8401ede36b1f2efca33de624c82ad3ec7